src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,26 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+
+    private final Comparator<Employee> employeeComparator = new Comparator<>() {
+        @Override
+        public int compare(Employee user1, Employee user2) {
+            if (user1.getAge() != user2.getAge()) {
+                return Integer.compare(user1.getAge(), user2.getAge());
+            } else {
+                return user1.getName().compareTo(user2.getName());
+            }
+        }
+    };
+
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Set<Employee> eployeSet = new TreeSet<>(employeeComparator);
```

Comment:
There is a typo in the variable name `eployeSet`. It should be corrected to `employeeSet` for consistency and clarity.